components/atom/tag/src/Actionable/Container.js

```diff
@@ -18,6 +25,7 @@ ActionableTagContainer.propTypes = {
   Link: PropTypes.func,
   href: PropTypes.string,
   target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top']),
+  rel: PropTypes.oneOf(['nofollow', 'noopener', 'noreferrer']),
```

Comment:
To make `atom/tag` a more reusable component, does it make sense to add all values `rel` can take?
I mean values like `author`,  `help`, `search`,  and of course, `tag`.

Reply:
Sure, I can add this linkTypes too